objectContainersHomework/Test/BazaDate/HobbyTest.java

```diff
@@ -0,0 +1,26 @@
+package BazaDate;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HobbyTest {
+
+    @Test
+   void  if_hobby_does_contains_only_country_return_hobby_and_country() {

```

Comment:
CE testezi tu aici? Ca list.add merge corect. List pe care il definesti in corpul metodei de test?